magento/pwa-studio
# [Docs] Venia Targets reference doc

## Description

This PR updates the JSDoc bocks in the `venia-ui-declare.js` file.
It also creates a file in devdocs that pulls this information into a reference-style topic.

## Related Issue

Closes PWA-647

## Acceptance 

@zetlen 

### Verification Stakeholders

@zetlen 

### Specification



### Verification Steps

1. Navigate to the pwa devdocs directory: `cd pwa-devdocs`
1. Start the HTML preview server: `yarn develop`
1. Verify file is auto-generated: `src/_includes/auto-generated/venia-ui/lib/targets/venia-ui-declare.md`
1. On the devdocs preview site, in the top navigation, click on **Venia UI API** under the _Reference API_ item.
1. Verify the Extensibility Targets topic appears with content.
1. Verify this topic is listed on the left navigation for this section

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
